app/scripts/modules/core/pipeline/config/triggers/travis/travisTriggerOptions.component.ts

```diff
@@ -65,17 +65,22 @@ export class TravisTriggerOptions {
   };
 }
 
+class TravisTriggerOptionsComponent implements IComponentOptions {
+  public bindings: any;
+  public templateUrl: string;
+  public controller: any;
+
+  constructor() {
+    this.bindings = {
+      command: '='
+    };
+    this.templateUrl = require('./travisTriggerOptions.component.html');
+    this.controller = TravisTriggerOptions;
+  }
+}
```

Comment:
Instead of putting this into a constructor, you should be able to just do this:
```typescript
class TravisTriggerOptionsComponent implements IComponentOptions {
  public bindings: any = {
    command: '='
  };
  public template = `<travis-trigger-options command="vm.command"></travis-trigger-options>`;
  public controller: any = TravisTriggerOptionsController;
}
```
(Want to rename the controller class from `TravisTriggerOptions` to `TravisTriggerOptionsController`, and you can omit the separate template file, since it's a single line)